The world 'parotTrouble' differs:
parotTrouble(true,7) returned Exception scala.MatchError: (true,7) (of class scala.Tuple2$mcZI$sp) while false were expected.
parotTrouble(false,6) returned Exception scala.MatchError: (false,6) (of class scala.Tuple2$mcZI$sp) while false were expected.

------------------------------------------
